### PR TITLE
feat(daffio): make header sticky on all pages

### PR DESCRIPTION
**Pull request by damienwebdev**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the new behavior?
This makes the header sticky. It depends on #420 and #426 

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

**Comment by damienwebdev** [comment]
@xelaint this PR should be working appropriately now that #426 is merged.

**Comment by xelaint** [comment]
1. There should be way for the header in the `docs` page to be uncontained while containing rest of pages in daffio.
2. HostListener to add drop-shadow on scroll.

File: apps/daffio/src/app/core/header/component/header.component.html [hunk 1]
```diff
@@ -1,16 +1,15 @@
-<daff-container size="md"> 
-  <daff-navbar class="navbar">
-    <div class="daffio-header__logo">
-      <ng-content select="[daffio-logo]"></ng-content>
+<daff-navbar class="navbar">
```
Comment (xelaint) [inline apps/daffio/src/app/core/header/component/header.component.html]: `.daffio-header__navbar`

File: apps/daffio/src/app/core/header/component/header.component.scss [hunk 2]
```diff
@@ -2,58 +2,61 @@
 @import 'theme';
 
 .navbar {
```
Comment (xelaint) [inline apps/daffio/src/app/core/header/component/header.component.scss]: .daffio-header__navbar

**Pull request closed without merging**